src/lightning/pytorch/_graveyard/ipu.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from typing import Any
+
+import lightning.pytorch as pl
+
+
+def _patch_sys_modules() -> None:
+    self = sys.modules[__name__]
+    sys.modules["lightning.pytorch.accelerators.ipu"] = self
+    sys.modules["lightning.pytorch.strategies.ipu"] = self
+    sys.modules["lightning.pytorch.plugins.precision.ipu"] = self
+
+
+class IPUAccelerator:
+    def __init__(self, *_: Any, **__: Any) -> None:
+        raise NotImplementedError(
+            "The `IPUAccelerator` class has been moved to an external package."
+            " Install the extension package as `pip install lightning-graphcore`"
+            " and import with `from lightning_graphcore import IPUAccelerator`."
+            " Please see: https://github.com/Lightning-AI/lightning-Graphcore for more details."
+        )
+
+
+class IPUStrategy:
+    """Strategy for training on single IPU device."""
```

Comment:
```suggestion
```

Reply:
Any reason to remove it?

Reply:
This isn't a strategy for training on single IPU device. It's a fake shim to avoid import errors that will raise an exception immediately